File: protenium-iro-hub/app/src/main/java/com/protenium/irohub/base/BaseViewModel.java
package com.protenium.irohub.base;

import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;


public class BaseViewModel extends ViewModel {
    private MutableLiveData<Boolean> isLoading = new MutableLiveData<>();


    public MutableLiveData<Boolean> getIsLoading() {
        return isLoading;
    }

    public void setIsLoading(Boolean isLoading) {
        this.isLoading.postValue(isLoading);
    }

    /*public String getCustomerId() {
        return SharedPrefs.getString(SharedPrefs.Keys.CUSTOMER_ID, "");
    }*/
}

File: propertyuser/app/src/main/java/com/property/propertyuser/base/BaseViewModel.kt
package com.property.propertyuser.base

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

open class BaseViewModel:ViewModel() {

    val isLoading: LiveData<Boolean> = MutableLiveData()

    fun setLoading(load: Boolean) {
        isLoading as MutableLiveData
        isLoading.value = load
    }
}